contentcuration/contentcuration/frontend/channelEdit/views/CurrentTopicView.vue

```diff
@@ -327,6 +327,20 @@
           }
           return false;
         },
+        ancestors() {
```

Comment:
What was your motivation to place this between the `selectAll` getter and setter?